packages/lab/src/stepper/Step.Connector.css

```diff
@@ -0,0 +1,52 @@
+.saltStepConnector {
+  --saltStep-connector-borderWidth: 2px;
+
+  grid-area: connector;
+  min-height: var(--salt-spacing-300);
+
+  transition-duration: inherit;
+  transition-timing-function: inherit;
+  transition-property: opacity, min-height;
+}
+
+.saltStepper-horizontal .saltStepConnector {
+  /* Distance between Icon and Connector */
+  --saltStep-connector-horizontal-gap: var(--salt-spacing-100);
+
+  position: absolute;
+  top: calc(var(--icon-size) / 2 - var(--saltStep-connector-borderWidth));
+  left: calc(50% + calc(var(--icon-size) / 2 + var(--saltStep-connector-horizontal-gap)));
+  right: calc(-50% + calc(var(--icon-size) / 2 + var(--saltStep-connector-horizontal-gap)));
+
+  border-top-width: var(--saltStep-connector-borderWidth);
```

Comment:
Local tokens should start with `--componentName`, CSS API variables should start with `--saltComponentName`, you might want to do something like `border-top-width: var(--saltStep-connector-borderWidth, --step-connector-borderWidth);`